### PR TITLE
add function logging

### PR DESCRIPTION
**Pull request by almottier**

### CODE REVIEW

File: logs/lib/getLogs.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+
+module.exports = {
+
```
Comment (AmFlint) [inline logs/lib/getLogs.js:6]: Run the linter please, and configure it on your local environment

File: logs/lib/getLogs.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      .then(() => this.getNamespaceFromList(this.namespaceName))
+      .then((namespace) => this.listFunctions(namespace.id))
+      .then(this.getApplicationId)
+      .then(this.getLines)
+      .then(this.printLines)
+  },
+
+  getApplicationId(functions){
+    for (const fn of functions) {
+        if (fn.name == this.options.f) {
```
Comment (AmFlint) [inline logs/lib/getLogs.js:18]: this.options.f => What is it ? Use a more complete variable name please

Comment (almottier) [reply logs/lib/getLogs.js:18]: this is the function name passed with logs command:
`logs -f {function_name}`

Comment (AmFlint) [reply logs/lib/getLogs.js:18]: Yes I meant to rename this parameters to `functionName` for example (f is a bit too short, not easy to understand what it is)

**Comment by AmFlint** [comment]
Please add an option `-c | --container-name` so our users can read logs from their containers too

File: logs/scalewayLogs.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+const BbPromise = require('bluebird');
+const setUpDeployment = require('../shared/setUpDeployment');
+const getLogs = require('./lib/getLogs');
+const { Api } = require('../shared/api');
+
+class ScalewayDeploy {
```
Comment (AmFlint) [inline logs/scalewayLogs.js:6]: This class should be called ScalewayLogs

File: logs/scalewayLogs.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    const api = new Api(credentials.apiUrl, credentials.token);
+
+    Object.assign(
+      this,
+      setUpDeployment,
+      getLogs,
+      api,
+    );
+
+
```
Comment (AmFlint) [inline logs/scalewayLogs.js:23]: 2 blank lines

File: logs/lib/getLogs.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const BbPromise = require('bluebird');
+
+module.exports = {
+
+  getLogs() {
+    return BbPromise.bind(this)
+      .then(() => this.getNamespaceFromList(this.namespaceName))
+      .then((namespace) => this.listFunctions(namespace.id))
```
Comment (AmFlint) [inline logs/lib/getLogs.js:10]: Also list containers please

File: logs/lib/getLogs.js (hunk 2 above)
Comment (almottier) [reply logs/lib/getLogs.js]: `-f` is required by serverless framework

**Pull request merged**